### ISSUE TITLE
`enforce-shorthand-classes` is not working if there is multiple sizes of different selector

### ISSUE THREAD

**Issue by JeromeDeLeon** [schoero/eslint-plugin-better-tailwindcss#204]
Hi, I was trying to migrate to the latest ShadCN combobox, and I found out that this is not reporting the lint rule for shorthand.

```diff
- const className = 'h-5 w-5 [&_data-slot=command-input-wrapper]:h-12 [&_data-slot=command-input-wrapper]:w-12';
+ const className = 'size-5 [&_data-slot=command-input-wrapper]:size-12';
```

If you want the exact classes for testing, here it is:

```diff
const className = `
**:data-[slot=command-input-wrapper]:h-12
[&_[cmdk-group-heading]]:px-2
[&_[cmdk-group-heading]]:font-medium
[&_[cmdk-group-heading]]:text-muted-foreground
[&_[cmdk-group]]:px-2
[&_[cmdk-group]:not([hidden])_~[cmdk-group]]:pt-0
- [&_[cmdk-input-wrapper]_svg]:h-5
- [&_[cmdk-input-wrapper]_svg]:w-5
+ [&_[cmdk-input-wrapper]_svg]:size-5
[&_[cmdk-input]]:h-12
[&_[cmdk-item]]:px-2
[&_[cmdk-item]]:py-3
- [&_[cmdk-item]_svg]:h-5
- [&_[cmdk-item]_svg]:w-5
+ [&_[cmdk-item]_svg]:size-5
`;
```

**Comment by JeromeDeLeon** [comment]
This is now working. Thank you so much ❤️
